vtm/src/org/oscim/map/Animator.java

```diff
@@ -220,38 +237,129 @@ public void animateZoom(long duration, double scaleBy,
         mPivot.x = pivotX;
         mPivot.y = pivotY;
 
-        animStart(duration, ANIM_SCALE, easingType);
+        animEaseStart(duration, ANIM_SCALE, easingType);
     }
 
+    /**
+     * Use animateEaseScroll instead
+     */
+    @Deprecated
     public void animateFling(float velocityX, float velocityY,
                              int xmin, int xmax, int ymin, int ymax) {
+        animateEaseScroll(velocityX, velocityY, xmin, xmax, ymin, ymax);
+    }
```

Comment:
No need to deprecate anything yet, it's all about choices.
Also old methods should remain "as is" for public use and add new ones separately.

Reply:
The method itself isn't deprecated. But the name is misleading. Isn't a fling at all! It's an ease animation for scroll/move. So I think this shouldn't used...

Reply:
It's a kinetic scroll somehow (probably not perfect), isn't it? I'd avoid breaking public API if possible.
BTW the "ease" word is not very recognizable, most won't understand what means, we can avoid it(?).

Reply:
> It's a kinetic scroll somehow (probably not perfect), isn't it? I'd avoid breaking public API if possible.

Ok, agreed.

> BTW the "ease" word is not very recognizable, most won't understand what means, we can avoid it(?).

Sure, but didn't find a better description for "ease" animations :| Any proposals?

Reply:
How about simply `kineticScroll` as the new `animateFling`?

Reply:
What `Fling` do you mean? The one with "ease" or the "physical" fling?
I wanted to keep calling physical as "fling" cause Android uses this as term.

Reply:
Meant to rename `animateEaseScroll` to `kineticScroll`, as there is already none "fling" in the name.

Reply:
Ah there is also the new `animateFlingScroll`, what's their difference?

Reply:
`animateEaseScroll` seems a changed `animateFling` and at the same time have a new `animateFlingScroll`.
So can keep old public `animateFling` as is and just add the new `animateFlingScroll` in `Animator2`?